### PR TITLE
Instance_identifiers

### PR DESCRIPTION
**Pull request by natalyapik**
create new derived metadb table instance_identifiers

### CODE REVIEW

File: sql_metadb/derived_tables/instance_identifiers.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
+-- create derived table to extract instance identifiers and value 
+
+DROP TABLE IF EXISTS instance_identifiers;
+
+CREATE TABLE instance_identifiers AS
+SELECT
+    instances.id AS instance_id,
+    jsonb_extract_path_text(instances.jsonb, 'hrid') AS instance_hrid,
+    jsonb_extract_path_text(idents.jsonb, 'identifierTypeId') AS identifier_type_id,
```
Comment (stdombek) [inline sql_metadb/derived_tables/instance_identifiers.sql:9]: "identifier_type_id" should be of data type uuid.

**Review by stdombek — CHANGES_REQUESTED** [review]
- [X] Query runs
- [X] Data output makes sense
- [ ] Data types look correct
- [X] Query logic seems correct
- [X] Column names make sense and are spelled correctly
- [X] Query syntax and style are appropriate (e.g., correct use of double quotes, lower case for data types)

For derived tables:
- [X] query has indexes on columns
- [X] vacuum analyze has been added to the bottom
- [X] table appears in runlist.txt at an appropriate point

File: sql_metadb/derived_tables/instance_identifiers.sql [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+CREATE INDEX ON instance_identifiers (instance_hrid);
+
+CREATE INDEX ON instance_identifiers (identifier_type_id);
+
+CREATE INDEX ON instance_identifiers (identifier_type_name);
+
+CREATE INDEX ON instance_identifiers (identifier);
+
+CREATE INDEX ON instance_identifiers (identifier_ordinaliry);
```
Comment (adoerrer) [inline sql_metadb/derived_tables/instance_identifiers.sql:31]: There's a little type in the column name. It should be: `identifier_ordinality`

**Review by adoerrer — CHANGES_REQUESTED** [review]
Thanks! Looks good but there're two minor changes to  do.
- [x] Query runs
- [x] Data output makes sense
- [] Data types look correct
- [x] Query logic seems correct
- [x] Column names make sense and are spelled correctly
- [x] Query syntax and style are appropriate (e.g., correct use of double quotes, lower case for data types)
For derived tables:
- [x] query has indexes on columns
- [x] vacuum analyze has been added to the bottom
- [x] table appears in runlist.txt at an appropriate point and with correct file name

**Review by stdombek — APPROVED** [review]
All queries:
- [X] Pull request has a complete description
- [X] Query runs without errors
- [X] Query output is correct
- [X] Query logic is clear and well documented
- [X] Query is readable and properly indented
- [X] Table and column names are in all-lowercase
- [X] Quotation marks are used only where necessary

Derived tables:
- [X] Query begins with user documentation in comment lines
- [X] File name is listed in `runlist.txt` after dependencies
- [X] All columns have indexes
- [X] Table is vacuumed and analyzed

PR has a very minimal description, but possible.

**Review by adoerrer — APPROVED** [review]
Thank you, Natalya. Looks and runs great.

**Pull request merged**